### ISSUE TITLE
Fails to build with sphinx 1.3

### ISSUE THREAD

**Issue by Ganeti-Issues-Migrator** [ganeti/ganeti#1175]
Originally reported of Google Code with ID 1119.

```
For details and proposed solutions, see https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=789394
```
Originally added on 2015-07-21 09:31:54 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-07-21 09:56:11 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
managed to reproduce the problem
```
Originally added on 2015-07-21 10:19:10 +0000 UTC.

**Comment by Ganeti-Issues-Migrator** [comment]
```
-- Empty comment --
```
Originally added on 2015-07-21 11:44:32 +0000 UTC.

Changed State: Started

**Comment by Ganeti-Issues-Migrator** [comment]
```
commit 268953c4cabdbed3f7f4959c01442fae8b132cc3
Author: Klaus Aehlig <aehlig@google.com>
Date:   Tue Jul 21 12:46:57 2015 +0200

    Support sphinx 1.3
    
    First, enable_manpages is now required to be a bool; fortunately,
    we set it via the environment in conf.py anyway, so no need to
    pass it as an option as well. Also, the default template has been
    renamed to classic; so branch on the sphinx version to choose the
    correct name. Fixes issue #1119.
    
    Signed-off-by: Klaus Aehlig <aehlig@google.com>
    Reviewed-by: Helga Velroyen <helgav@google.com>

```
Originally added on 2015-07-21 13:37:38 +0000 UTC.

Added Labels: Type-Enhancement
Removed Labels: Type-Defect
Changed State: Fixed
Added to Milestone: Release2.12
